contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -0,0 +1,119 @@
+# Scoped Kubelet API Access
+
+Author: Jordan Liggitt (jliggitt@redhat.com)
+
+## Overview
+
+Kubelets are responsible for:
+* creating and updating status of their Node API object
+* updating status of Pod API objects bound to their node
+* creating/deleting "mirror pod" API objects for statically-defined pods running on their node
+* reading Secret and ConfigMap objects referenced by pod specs bound to their node
```

Comment:
+1 to (attempting?) to make a semi-complete list, even if they're not in-scope for v1.7 or v1.8 -- be nice to see a what's *sufficient* to get where we want to be, not just *known-necessary*.

Also @jbeda mentioned that we should eventually make kubelet/nodes have whitelisted access to the specific set of things they need access to?  And if so, having that list of necessary permissions would be a pre-requisite?